mozilla/fx-private-relay
# Link to Chrome extension on Chrome

# New feature description

This adds pointers to the Chrome extension when viewing in Chrome. Fixes #1614, 

# Screenshot (if applicable)

![image](https://user-images.githubusercontent.com/4251/157241878-9a9829b2-b880-41a2-95eb-ee24ec697de5.png)

![image](https://user-images.githubusercontent.com/4251/157241786-02b73535-a541-4842-bc27-41d1719c5043.png)

# How to test

With an account that just updated to Premium:

- Open the onboarding's final step in a browser that supports Chrome extensions. You should see a link to the Chrome extension.
- Open the onboarding's final step in Firefox. You should see a link to the Firefox extension.
- Open the onboarding's final step in a browser for which we don't have an extension. You should see no links to extensions. (**Important:** I wasn't able to test this myself as FxA didn't work with the only browsers I have that fits this description.)

Then with any account that is not in the onboarding, visit the dashboard:

- On Firefox, you should see a banner to install the Firefox add-on.
- On a browser that supports the Chrome extension, you should see a banner to install that. There should be no banner to install Firefox.
- On another browser, you shouldn't see a banner. (**Important:** I wasn't able to test this myself as FxA didn't work with the only browsers I have that fits this description.)

# Checklist

- [ ] l10n dependencies have been merged, if any. - Not yet: https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/65
